app/DoctrineMigrations/Version20191015091033.php

```diff
@@ -0,0 +1,30 @@
+<?php declare(strict_types=1);
+
+namespace Application\Migrations;
+
+use Doctrine\DBAL\Schema\Schema;
+use Doctrine\Migrations\AbstractMigration;
+
+/**
+ * Auto-generated Migration: Please modify to your needs!
+ */
+final class Version20191015091033 extends AbstractMigration
+{
+    public function up(Schema $schema) : void
+    {
+        // this up() migration is auto-generated, please modify it to your needs
+        $this->abortIf($this->connection->getDatabasePlatform()->getName() !== 'mysql', 'Migration can only be executed safely on \'mysql\'.');
+
+        $this->addSql('ALTER TABLE team_membership ADD isSuspended TINYINT(1) NOT NULL');
+        $this->addSql('UPDATE team_membership SET isSuspended=0');
+        $this->addSql('ALTER TABLE team_membership MODIFY isSuspended TINYINT(1) NOT NULL');
```

Comment:
```sql
ALTER TABLE team_membership ADD isSuspended TINYINT(1) NOT NULL
```
Her må vel tabellen starte med å være NULLABLE, hvis ikke får vi problemer med eksisterende rader i tabellen?
```sql
ALTER TABLE team_membership ADD isSuspended TINYINT(1)
```
Og så blir `NOT NULL` satt i MODIFY til slutt.

---

En alternativ måte å gjøre det på er å sette en default value. Da holder det med én query:
```sql
ALTER TABLE team_membership MODIFY isSuspended TINYINT(1) NOT NULL DEFAULT 0
```